test/bin/build_images.sh

```diff
@@ -6,6 +6,7 @@
 set -euo pipefail
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=/dev/null
```

Comment:
Why this instead of properly configuring shellcheck to find the file?

Reply:
The problem is that shellcheck does not follow paths when variables are specified. I could not make it work in VSCode.

https://www.shellcheck.net/wiki/SC1090

Reply:
I don't think that problem with VSCode extension should be extended to `make verify-sh` thus affecting the CI.
Can you try changing setting of `shellcheck.customArgs`  to match args given to shellcheck in `verify-shell.sh` and see if that helps?

Reply:
Looking at [this code](https://github.com/openshift/microshift/blob/main/scripts/verify/verify-shell.sh#L26), we're already excluding the SC1091, so there is not shell-check error.

My fix only comes to address the issue inside the VSCode.

Reply:
@pmtk , I uploaded another fix. Is this what you had in mind?